### DISCUSSION TITLE
How to auto register a new user?

### DISCUSSION THREAD

**Discussion by pik33** [InnovateAsterisk/Browser-Phone#434] [Q&A]
I want to integrate the Browser Phone into a Wordpress site. I have the user registered in Wordpress, so I have the user name, etc. I also have the SIP server data etc. I can assign the unique telephone number for the user based on his Wordpress uid.

As it is now, the user has to manually enter the data which he doesn't even know into the initial Browser Phone window.

Is it possible to call Browser Phone when the user registered in Wordpress and pass the Browser Phone these parameters so the user doesn't need to do it manually?

**Comment by InnovateAsterisk** [comment]
You are looking for provisioning. It works like this;

Take a look at the sample index.html file, especially the phoneOptions variable declared there:
https://github.com/InnovateAsterisk/Browser-Phone/blob/12061962af0ba09899c432cdc1e18a2f2297740e/Phone/index.html#L50

Use this json object to populate the script with variables from here:
https://github.com/InnovateAsterisk/Browser-Phone/blob/master/Phone/phone.js#L477

If you add them to the global object, they are used, including nodes like ```options.SipPassword```
